activerecord/lib/active_record/railtie.rb

```diff
@@ -26,6 +26,7 @@ class Railtie < Rails::Railtie # :nodoc:
 
     config.active_record.use_schema_cache_dump = true
     config.active_record.maintain_test_schema = true
+    config.active_record.belongs_to_required_by_default = true
```

Comment:
Yeah, what I'm talking about is that subclasses of `ActiveRecord::Base` will have different configurations when used in the initializers if the values are set to `ActiveRecord::Base` only in the `after_initializer` block.